client/my-sites/plugins/plugin-activate-toggle/index.jsx

```diff
@@ -65,12 +64,9 @@ export class PluginActivateToggle extends Component {
 					className="plugin-activate-toggle"
 					htmlFor={ 'disconnect-jetpack-' + site.ID }
 					>
-					<DisconnectJetpackButton
-						disabled={ disabled || ! plugin }
-						site={ site }
-						redirect="/plugins/jetpack"
-						isMock={ isMock }
-						/>
+					<a href={ '/settings/general/' + site.slug }>
+						{ translate( 'Manage Connection', { context: 'manage Jetpack connnection settings link' } ) }
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?